templates/boilerplate/_dockerfiles/build-qx/Dockerfile

```diff
@@ -12,9 +12,10 @@ RUN npm install qxcompiler
 
 # TODO PC: why to create again? not better taking entire client-qx??
 ENV PATH="/home/node/node_modules/.bin/:${PATH}"
-RUN qx create qxapp -I --qxpath /home/node/node_modules/qooxdoo-sdk/framework/
+#RUN qx create qxapp -I --qxpath /home/node/node_modules/qooxdoo-sdk/framework/
 
-WORKDIR /home/node/qxapp
 VOLUME /home/node/qxapp
+WORKDIR /home/node/qxapp
+
 
-CMD ["qx", "compile"]
+ENTRYPOINT qx compile --watch
```

Comment:
This is fine, but for command line IMHO it is a better practice to express it as a list ...